tools/cli/commands/create.py

```diff
@@ -358,80 +383,108 @@ def flags(parser):
     return
 
 
-def create_network(args, gcloud_compute):
-    """Create the `datalab-network` network.
+def create_network(args, gcloud_compute, network_name):
+    """Create the specified network.
 
     Args:
       args: The Namespace returned by argparse
       gcloud_compute: Function that can be used for invoking `gcloud compute`
+      network_name: The name of the network
     Raises:
       subprocess.CalledProcessError: If the `gcloud` command fails
     """
     if utils.print_info_messages(args):
-        print('Creating the network {0}'.format(_DATALAB_NETWORK))
+        print('Creating the network {0}'.format(network_name))
     create_cmd = [
-        'networks', 'create', _DATALAB_NETWORK,
+        'networks', 'create', network_name,
         '--description', _DATALAB_NETWORK_DESCRIPTION]
     utils.call_gcloud_quietly(args, gcloud_compute, create_cmd)
     return
 
 
-def ensure_network_exists(args, gcloud_compute):
-    """Create the `datalab-network` network if it does not already exist.
+def ensure_network_exists(args, gcloud_compute, network_name):
+    """Create the specified network if it does not already exist.
 
     Args:
       args: The Namespace returned by argparse
       gcloud_compute: Function that can be used for invoking `gcloud compute`
+      network_name: The name of the network
     Raises:
       subprocess.CalledProcessError: If the `gcloud` command fails
     """
-    get_cmd = ['networks', 'describe', '--format', 'value(name)',
-               _DATALAB_NETWORK]
+    get_cmd = ['networks', 'describe', '--format', 'value(name)', network_name]
     try:
         utils.call_gcloud_quietly(
             args, gcloud_compute, get_cmd, report_errors=False)
     except subprocess.CalledProcessError:
-        create_network(args, gcloud_compute)
+        create_network(args, gcloud_compute, network_name)
     return
 
 
-def create_firewall_rule(args, gcloud_compute):
-    """Create the `datalab-network-allow-ssh` firewall rule.
+def create_firewall_rule(args, gcloud_compute, network_name, rule_name):
+    """Create the `{rule_name}` firewall rule.
```

Comment:
The previous comment included the rule name, from which one can understand in general what that rule does. SInce the new comment no longer includes that piece of information, it would be nice to add a word or two to the comment, such as 'Create the {rule_name} allow-ssh rule.'

Reply:
Done.